OpenRA.Mods.Common/Activities/Repair.cs

```diff
@@ -68,7 +68,7 @@ public override Activity Tick(Actor self)
 			if (host.Type == TargetType.Invalid || health == null)
 				return NextActivity;
 
-			if (closeEnough.LengthSquared > 0 && !host.IsInRange(self.CenterPosition, closeEnough))
+			if (closeEnough != WDist.Zero && !host.IsInRange(self.CenterPosition, closeEnough))
```

Comment:
WDist supports `>`, so you should revert the `!=` to make sure there won't be any unwanted behavior.